addon/components/cookie-banner/style.scss

```diff
@@ -0,0 +1,20 @@
+#cookie-banner {
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    background-color: #263947;
+    color: white;
+    display: flex;
+    align-items: center;
+    z-index: 1;
```

Comment:
This z-index seemed a little low to me and it definitely is:
<img width="359" alt="screen shot 2018-07-10 at 3 59 37 pm" src="https://user-images.githubusercontent.com/1322421/42534330-7c2cabba-845a-11e8-8c62-5e6234d20438.png">